lib/src/cli/flutter_cli.dart

```diff
@@ -474,7 +474,11 @@ final int _lineLength = () {
   }
 }();
 
-extension on TestEvent {
+// The extension is intended to be unnamed, but it's not possible to
+// due to an issue with Dart SDK 2.18.0.
+//
+// Once the min Dart SDK is bumped, this extension can be unnamed again.
```

Comment:
What is the p[roblem we have here? 👀

Reply:
@wolfenrain can give more context here.

See [this](https://github.com/VeryGoodOpenSource/very_good_cli/actions/runs/4365914878/jobs/7635200434).